tests/unit/ops/test_ops_schema.py

```diff
@@ -27,6 +27,12 @@
         ops.Normalize(),
         ops.TargetEncoding(["1"]),
         ops.AddMetadata(tags=["excellent"], properties={"domain": {"min": 0, "max": 20}}),
+        ops.AddTags(tags=["excellent"]),
+        ops.AddProperties(properties={"domain": {"min": 0, "max": 20}}),
+        ops.TagAsUserID(),
+        ops.TagAsItemID(),
+        ops.TagAsUserFeatures(),
+        ops.TagAsItemFeatures(),
```

Comment:
@jperez999 Is this sufficient to test that these operators add the tags we're expecting them to, or is there another place we need to add tests for that?

Reply:
no this is good. This is our test framework for testing all metadata implementations.